packages/manager/apps/pci-project/src/pages/home/components/DashboardTiles.component.tsx

```diff
@@ -29,38 +19,45 @@ export default function DashboardTiles() {
       role="region"
       aria-label={t('pci_projects_project_dashboard_tiles_aria')}
     >
-      {sections.map((section, sectionIdx) => (
-        <div
-          key={sectionIdx}
-          className="flex flex-col h-full"
-          role="region"
-          aria-label={t('pci_projects_project_section_aria', {
-            title: section.title,
-          })}
-        >
-          <div className="h-full [&>*]:h-full">
-            <ManagerTile>
-              {section.title && (
-                <>
-                  <ManagerTile.Title>{section.title}</ManagerTile.Title>
-                  <ManagerTile.Divider />
-                </>
-              )}
-              {section.items.map((item, itemIdx) => (
-                <TileItem
-                  key={`${sectionIdx}-${itemIdx}`}
-                  item={item}
-                  section={section}
-                  itemIdx={itemIdx}
-                  totalItems={section.items.length}
-                  isLoading={isLoading}
-                  projectId={projectId}
-                />
-              ))}
-            </ManagerTile>
+      {tiles
+        .filter((tile) =>
+          tile.type === 'billing'
+            ? tile.items.some((item) => !item.hideTileIfNoOtherItems)
+            : tile.items.length > 0,
+        )
+        .map((tile, tileIdx) => (
+          <div
+            key={tileIdx}
```

Comment:
Be careful because array index should not be used as a react key => https://react.dev/learn/rendering-lists#rules-of-keys